v2/pkg/operators/common/dsl/dsl.go

```diff
@@ -256,6 +363,11 @@ var functions = map[string]govaluate.ExpressionFunction{
 		gologger.Info().Msgf("print_debug value: %s", fmt.Sprint(args))
 		return true, nil
 	},
+	// is_before_now compares a timestamp and returns true if the first
+	// passed argument is a time.Time that has already passed.
+	"time_now": func(args ...interface{}) (interface{}, error) {
```

Comment:
the comment and the implementation are not in sync. Also, why would you transform it to a float?

Reply:
Removed this function and used `unixtime()` instead. As for why float, govaluate comparisons require integers to be float for it to work.